### ISSUE TITLE
travis build reports success even with an error

### ISSUE THREAD

**Issue by bcail** [loris-imageserver/loris#247]
See https://travis-ci.org/loris-imageserver/loris/builds/171507934 and download the raw log (https://api.travis-ci.org/jobs/171507935/log.txt?deansi=true). There's a test failure at the bottom.

**Comment by jpstroop** [comment]
Wow, that's a little scary. I just pulled the latest development revisions and see this:

```
FAILED (errors=1)
jstroop@ltopstroop1:~/workspace/loris (development)$ echo $?
0
```

I'll make a PR to see if having it return non-0 changes things.

**Comment by jpstroop** [comment]
Closed by #248
